images/fluentbit/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM debian:bullseye-slim as build-env
```

Comment:
How about add a Makefile like this: https://github.com/bentoml/yatai-deployment/blob/main/container_images/bento-debugger/Makefile#L3

Reply:
done